Makefile

```diff
@@ -1,5 +1,5 @@
 # Current Operator version
-VERSION ?= "$(shell git describe --tags | grep -Po "([\d\.]+)")" ## version, without the 'v' prefix
+VERSION ?= "$(shell git describe --tags | sed 's/^v//')"
```

Comment:
Shouldn't this be changed in a separate PR?

Reply:
Created #105